features/admin.actions.v1/pages/action-edit-page.tsx

```diff
@@ -0,0 +1,694 @@
+                    authProperties.value = values.valueAuthProperty;
+
+                    break;
+                case AuthenticationType.NONE:
+                    break;
+                default:
+                    break;
+            }
+        }
+
+        if (isCreating) {
+            const actionValues: ActionInterface = {
+                endpoint: {
+                    authentication: {
+                        properties: authProperties,
+                        type: authType
+                    },
+                    uri: values.endpointUri
+                },
+                name: values.name
+            };
+
+            setIsSubmitting(true);
+            createAction(actionTypeApiPath, actionValues)
+                .then(() => {
+                    handleSuccess(ActionsConstants.CREATE);
+                    fetchActionConfigurations();
+                })
+                .catch((error: AxiosError) => {
+                    handleError(error, ActionsConstants.CREATE);
+                })
+                .finally(() => {
+                    setIsSubmitting(false);
+                });
+        } else {
+            // Updating the action
+            const updatingValues: ActionUpdateInterface = {
+                endpoint: {
+                    authentication: isAuthUpdating ? {
+                        properties: authProperties,
+                        type: authType
+                    } : undefined,
+                    uri: values?.endpointUri
+                },
+                name: values?.name
+            };
+
+            setIsSubmitting(true);
+            updateAction(actionTypeApiPath, initialValues.id, updatingValues)
+                .then(() => {
+                    handleSuccess(ActionsConstants.UPDATE);
+                    setIsAuthUpdating(false);
+                    fetchActionConfigurations();
+                })
+                .catch((error: AxiosError) => {
+                    handleError(error, ActionsConstants.UPDATE);
+                })
+                .finally(() => {
+                    setIsSubmitting(false);
+                });
+        }
+    };
+
+    return !isActionDataLoading && actionData ? (
+        <PageLayout
+            title={ resolveActionTitle(actionTypeApiPath) }
+            description={ resolveActionDescription(actionTypeApiPath) }
+            backButton={ {
+                "data-componentid": `${ _componentId }-${ actionTypeApiPath }-page-back-button`,
+                onClick: () => handleBackButtonClick(),
+                text: t("console:manage.features.actions.goBackActions")
+            } }
+            bottomMargin={ false }
+            contentTopMargin={ true }
+            pageHeaderMaxWidth={ false }
+            data-componentid={ `${ _componentId }-${ actionTypeApiPath }-page-layout` }
+        >
+            { actionToggle() }
+            {
+                <Ref innerRef={ pageContextRef }>
```

Comment:
Any specific reason to use a `<Ref>` component here?

Reply:
Fixed with https://github.com/wso2/identity-apps/pull/6722/commits/d96314b591f6f6f33b709b0600c399b789e7fcef